db/migrate/20191118151439_create_topics.rb

```diff
@@ -0,0 +1,9 @@
+class CreateTopics < ActiveRecord::Migration[5.2]
+  def change
+    create_table :topics do |t|
+      t.column :name, :string, null: false
```

Comment:
```suggestion
      t.column :name, :string, null: false, index: true, unique: true
```
You can also do this

Reply:
Thank you for the comment, we try it but does not work; in the schema just show the index condition not the unique one.